test/simulation/docker-clean.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+docker-compose down
+docker volume rm simulation_btcd-vol
```

Comment:
I don't think it was related to the `temp` folder, but yes it is annoying.
I changed it now so that it won't return an error status, but the output still exist - I thought it would be useful to know when nothing was cleaned.